CrocoParallel.py

```diff
@@ -248,7 +248,7 @@ def prepare_offline_env(self, date, idate, mbdir, mb, mbdirs_forc):
         try:
             safe_create_link(self.options.forcing + '/' + mbdirs_forc + '/meteo/FORCING_' + date_begin_forc.strftime('%Y%m%d%H') + '_' + date_end_forc.strftime('%Y%m%d%H') + '.nc',
                              'FORCING.nc')
-        except IOError:
+        except:
```

Comment:
Did you delete `IOError` on purpose/what was the reason? Were you getting different exceptions?

Reply:
ok thanks for your explanation, I guess we can keep your version ;)